conf/minifi.properties

```diff
@@ -29,9 +29,11 @@ nifi.provenance.repository.max.storage.time=1 MIN
 nifi.provenance.repository.max.storage.size=1 MB
 nifi.flowfile.repository.directory.default=${MINIFI_HOME}/flowfile_repository
 nifi.flowfile.checkpoint.directory.default=${MINIFI_HOME}/flowfile_checkpoint
+# nifi.flowfile.repository.rocksdb.compression=auto
```

Comment:
Is it possible to change these settings without losing existing repository data? So basically does it detect existing data in a different format and do conversion at startup? If not, is this something we could make work, possibly in a followup?

Reply:
I tried switching compression types between restarts and the flow continued where it left off previously. I could see in the rocksdb log files the used compression was changed to the new one in the minifi property file so it seems to me that rocksdb could handle the config change.